blog/2024/terraform-provider-adoption.adoc

```diff
@@ -0,0 +1,40 @@
+:title: Keycloak Adopts Terraform Provider
+:date: 2024-12-02
+:publish: false
+:author: Thomas Darimont
+
+# New Repository
+
+We're excited to announce that the Keycloak Terraform Provider has officially moved under the Keycloak organization! 
+You can find the new repository https://github.com/keycloak/terraform-provider-keycloak[here].
```

Comment:
Strictly speaking, its not a new repository. Maybe "the new repository location"?

Reply:
Fixed